atomic_reactor/plugins/post_pulp_sync.py

```diff
@@ -236,5 +245,13 @@ def run(self):
             for image_name in images:
                 self.log.info("image available at %s", image_name.to_str())
 
+        # Fetch the repository content so we can remove v2 schema 2
+        # manifests from Koji metadata if they are not present
+        # (i.e. if Pulp does not have v2 schema 2 support).
```

Comment:
After a sync, can we reliably use the presence of v2 schema 2 content to determine pulp support?
If so, may we can expose this information to `koji_upload` instead of using the plugin arg `report_multiple_digests` ?

Reply:
The sync doesn't happen until after `koji_upload` runs.

Reply:
bummer..

Reply:
That's a fine idea for koji_promote though. Implemented.